packages/mdc-chips/_mixins.scss

```diff
@@ -611,3 +612,19 @@ $mdc-chip-ripple-target: ".mdc-chip__ripple";
     }
   }
 }
+
+///
+/// Sets the density scale for chips.
+///
+/// @param {Number | String} $density-scale - Density scale value for component.
+///     Supported density scale values are `-2`, `-1`, `0`.
+///
+@mixin mdc-chip-density($density-scale, $query: mdc-feature-all()) {
+  $height: mdc-density-prop-value(
+    $density-config: $mdc-chip-density-config,
+    $density-scale: $density-scale,
+    $property-name: height,
+  );
+
+  @include mdc-chip-height($height, $query: $query);
```

Comment:
Note: we also need to reset the touch target by default in density mixin, but this can be done in a separate PR.

Reply:
Yep, will do in a followup PR.